smarts/core/remote_agent.py

```diff
@@ -77,9 +77,6 @@ def terminate(self):
         if (self._act_future is not None) and (not self._act_future.done()):
             self._act_future.cancel()
 
-        # Close worker channel
-        self._worker_channel.close()
```

Comment:
Would another alternative (such the "normal" non-crashing case still works)  be to wrap this in a try block and just swallow any exceptions that happen?

Reply:
I tested (i) normal non-crashing case and (ii) crashing case. Both appears to work fine with the latest code changes.

I initially thought of using a `try-except` block to hide the error messages. Hence, the name of this branch `try-except-zoo-manager`. Multiple attempts to close a process which was already closed, resulted in error messages being thrown. So, instead of hiding the error, I thought it was better to prevent the multiple attempts from occurring.